src/mainwin.h

```diff
@@ -1,7 +1,7 @@
 #ifndef __MAINWIN_H__
 #define __MAINWIN_H__
 
-cairo_surface_t * pixsurf;
+extern cairo_surface_t * pixsurf;
```

Comment:
Hmm... But it should be also defined in some source file. `extern` just declares it. I think it wasn't defined in any source file before.